### PR TITLE
CI/CLN: Lint more class references in tests

### PR DESCRIPTION
**Pull request by dsaxton**

### CODE REVIEW

**Comment by dsaxton** [comment]
> I take it this is a get-it-all-done-at once PR?

It started out as just MultiIndex, then I may have gotten a bit carried away

**Comment by jbrockmendel** [comment]
needs rebase

File: ci/code_checks.sh [hunk 1]
```diff
@@ -146,7 +146,7 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
     MSG='Check for inconsistent use of pandas namespace in tests' ; echo $MSG
-    for class in "Series" "DataFrame" "Index"; do
+    for class in "Series" "DataFrame" "Index" "MultiIndex" "Timestamp" "Timedelta" "TimedeltaIndex" "DatetimeIndex" "Categorical"; do
```
Comment (jreback) [inline ci/code_checks.sh:149]: didn't we move these to pre-commit?

**Review by jreback — CHANGES_REQUESTED** [review]
ok will have to merge this right after you rebase to avoid conflicts

File: ci/code_checks.sh (hunk 1 above)
Comment (dsaxton) [reply ci/code_checks.sh:149]: I don't think that's been done yet but @MarcoGorelli seemed to think it was doable

Comment (MarcoGorelli) [reply ci/code_checks.sh:149]: On my to-do list :innocent:  I hope to get round to it next week

**Comment by dsaxton** [comment]
@jreback This is rebased and almost green if we want to merge and avoid more conflicts

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @dsaxton

**Pull request merged**